cypress/e2e/voucher/confirm-your-name.cy.js

```diff
@@ -1,6 +1,6 @@
 describe('Confirm your name', () => {
     beforeEach(() => {
-        cy.visit('/fixtures/voucher?redirect=/confirm-your-name');
+        cy.visit('/fixtures/voucher?redirect=/confirm-your-name&progress=skip');
```

Comment:
Was this supposed to be `progress=` to match up with the changes in voucher fixtures?

Reply:
nope, purposefully using one that doesn't exist so it doesn't do the sharecode sending

Reply:
I was using one that didn't exist to skip the behaviour of it being unset, but I've rewritten how it works to maybe be more obvious